packages/terra-site/src/examples/paginator/SearchPaginatorExample.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Dialog from 'terra-dialog';
+import Paginator from 'terra-paginator/lib/Paginator';
+
+const totalCount = 450;
+
+const fillArray = (value, len) => {
+  const arr = [];
+  for (let i = 0; i < len; i += 1) {
+    arr.push(<p key={Math.floor(Math.random() * Math.floor(100000))}>{value}</p>);
```

Comment:
Do you need Math.floor on 100,000? Feels like the base Math.floor does all the rounding for you.

Reply:
It is just to increase the variance in possible numbers for the key to avoid duplicates. This doesn't need to be optimized.